Marlin/example_configurations/BQ/Hephestos_2/Configuration.h

```diff
@@ -44,14 +44,6 @@
 //===========================================================================
 
 /**
- * BQ Hephestos 2 Configuration
- *
- * This configuration supports the standard Hephestos 2 with or without the
- * heated bed kit featured at https://store.bq.com/en/heated-bed-kit-hephestos2
- *
- * Enable HEPHESTOS2_HEATED_BED_KIT in Configuration_adv.h for all functionality
- * related to the heated bed.
- *
```

Comment:
This comment may be helpful to leave in, along with leaving the `HEPHESTOS2_HEATED_BED_KIT` disabled by default.